### PR TITLE
Sort alphanumerically using numeric option of collator

### LINKED ISSUE #459: Sort legend alphanumerically
Current legend doesn't consider numerical order, making the legend start with numbers sorted unexpectedly. Probably can be solved with : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/numeric 

Problem: 
Screen Shot 2023-02-15 at 10 46 30 AM

### PR DESCRIPTION
**Pull request by hanbyul-here**
Close #459 
Legend gets sorted  `a1,a11, a2` right now. 
This PR should fix this problem.

### CODE REVIEW

File: app/scripts/components/common/chart/utils.ts [hunk 1]
```diff
@@ -92,8 +92,11 @@ export function getFData({
   yKey: string;
   dateFormat: string;
 }): { uniqueKeys: any[], fData: FormattedTimeSeriesData[] } {
+  // Check sensitivity value
+  const collator = new Intl.Collator(undefined, {numeric: true, sensitivity: 'base'});
```
Comment (hanbyul-here) [inline app/scripts/components/common/chart/utils.ts:96]: Anybody can give me insight if it is reasonable to consider accent or other diacritic marks for sorting? (or any other values for sensitivity? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator. options are `base`, `case`, `variant`) I've been living accentless life so I have no sense.

Comment (danielfdsilva) [reply app/scripts/components/common/chart/utils.ts:96]: I imagine that for the majority of use cases we don't need to worry about accents. However, I'd say to disregard accents when sorting. Consider `à` equal to `a`

File: app/scripts/components/common/chart/utils.test.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import { getFData } from '$components/common/chart/utils';
```
Comment (danielfdsilva) [inline app/scripts/components/common/chart/utils.test.ts:1]: yay tests! 🎉

File: jest.config.js [hunk 3]
```diff
@@ -93,7 +93,8 @@ module.exports = {
         : acc;
     }, {}),
     '.+\\.(css|styl|less|sass|scss)$':
-      '<rootDir>/node_modules/jest-css-modules-transform'
+      '<rootDir>/node_modules/jest-css-modules-transform',
+    d3: '<rootDir>/node_modules/d3/dist/d3.min.js'
```
Comment (danielfdsilva) [inline jest.config.js:97]: why was it needed to specify d3?

Comment (hanbyul-here) [reply jest.config.js:97]: `chart/utils` imports d3 and jest has problems importing packages without `exports.require` option declared (https://github.com/facebook/jest/issues/12036#issuecomment-1039479569)

Comment (danielfdsilva) [reply jest.config.js:97]: gotcha! 👍

**Comment by hanbyul-here** [comment]
Flagging that https://github.com/NASA-IMPACT/veda-config/pull/193 <- legend for this data should be reconfigured with this change. (It should be formatted as `20% <` )

**Pull request merged**